web/src/features/hfiCalculator/components/DailyViewTable.tsx

```diff
@@ -31,7 +37,9 @@ const useStyles = makeStyles({
     width: '100%'
   },
   tableContainer: {
-    maxHeight: 480
+    maxHeight: 1080,
+    maxWidth: 1900,
+    minWidth: 1600
```

Comment:
You can add
`& .MuiTableCell-head': {
     fontWeight: 'bold',
    padding: '1px'
} ` and then get rid of tableHeader